source/plugins/ruby/constants.rb

```diff
@@ -56,9 +64,11 @@ class Constants
     CPU_USAGE_MILLI_CORES = "cpuUsageMillicores"
     MEMORY_WORKING_SET_BYTES= "memoryWorkingSetBytes"
     MEMORY_RSS_BYTES = "memoryRssBytes"
+    PV_USED_BYTES = "pvUsedBytes"
     DEFAULT_MDM_CPU_UTILIZATION_THRESHOLD = 95.0
     DEFAULT_MDM_MEMORY_RSS_THRESHOLD = 95.0
     DEFAULT_MDM_MEMORY_WORKING_SET_THRESHOLD = 95.0
+    DEFAULT_MDM_PV_UTILIZATION_THRESHOLD = 60.0
```

Comment:
curios, why default PV utilization threshold is slow low compared other metrics, for example, memory and cpu  where we have 95% threshold.

Reply:
I believe the default threshold was decided by @vishiy and Sahil

Reply:
As an update after a meeting yesterday - the threshold was recommended by the storage team since PVs may reach capacity more quickly. We might change the default later after seeing the initial telemetry